src/main/java/org/cloudburstmc/proxypass/network/bedrock/session/UpstreamPacketHandler.java

```diff
@@ -243,8 +243,8 @@ private void initOnlineLoginChain(StepMCChain.MCChain mcChain) throws Exception
             initMojangPublicKey();
         }
 
```

Comment:
You should use a permalink here else the line numbers will point into random code when I update that class in the future